### PR TITLE
Coven

### PR DESCRIPTION
**Pull request by keepermanj**
Witches coven:

http://keeperrl.com/wiki/index.php?title=Requests_for_new_configurations#The_seven_witches

### CODE REVIEW

**Comment by keepermanj** [comment]
New code was being added in the same places as the demons. Conflicts resolved by allowing it to add the new code into different places (which it is happy to do).

**Comment by miki151** [comment]
Witches are a bit weak, so make it a lesser enemy. Also I think they would look better in a village, like SettlementType::SMALL_VILLAGE.

Btw, there are a bunch of tabs in the changes.

**Comment by SoftMonster** [comment]
I'll try lesser villain with small village.

**Comment by SoftMonster** [comment]
I changed my editor to use spaces instead of tabs.

**Comment by SoftMonster** [comment]
I took a look at the small village. It isn"t bad. It did throw in a shopkeeper I didn't really want. The loot was also a lot smaller. I can upload this version if you like or try something else.

File: creature_factory.cpp [hunk 1]
```diff
@@ -1208,7 +1212,7 @@ CreatureAttributes CreatureFactory::getAttributesFromId(CreatureId id) {
           c.name->setFirst(NameGenerator::get(NameGeneratorId::DRAGON)->getNext());
           c.spells->add(SpellId::HEAL_SELF);
           c.spells->add(SpellId::CURE_POISON);
-          c.spells->add(SpellId::DECEPTION);
+          //c.spells->add(SpellId::DECEPTION);  Dragon hostile to its own illusions
```
Comment (miki151) [inline creature_factory.cpp]: Please uncomment this.

File: creature_factory.cpp [hunk 2]
```diff
@@ -1224,7 +1228,7 @@ CreatureAttributes CreatureFactory::getAttributesFromId(CreatureId id) {
           c.permanentEffects[LastingEffect::RANGED_VULNERABILITY] = 1;
           c.spells->add(SpellId::HEAL_SELF);
           c.spells->add(SpellId::CURE_POISON);
-          c.spells->add(SpellId::DECEPTION);
+          //c.spells->add(SpellId::DECEPTION);  Dragon hostile to its own illusions
```
Comment (miki151) [inline creature_factory.cpp]: Please uncomment this.

**Comment by miki151** [comment]
The shopkeeper is there because you defined shopFactory. But you can leave them in the cave for now, I'll take a look later and change it to what I like more.

**Comment by SoftMonster** [comment]
Moddable

**Comment by keepermanj** [comment]
Have a witches tribe with medusa leader kn mod already.

**Pull request closed without merging**